### PR TITLE
Create helpers.js

### PR DESCRIPTION
**Pull request by JustinyAhin**
Utility functions for Core e2e tests

### CODE REVIEW

**Comment by JustinyAhin** [comment]
cc @kevin940726

**Comment by kevin940726** [comment]
I think we might have a misunderstanding here. I didn't mean to rewrite testing utilities in Core, we should be able to continue using utilities from gutenberg, but rather to rewrite them in gutenberg and try to make them faster.

Take `isThemeInstalled` for example. You did a great job here re-implementing it using REST API for maximum performance. While the [original code](https://github.com/WordPress/gutenberg/blob/update/remove-styles-link/packages/e2e-test-utils/src/theme-installed.js) in gutenberg still requires several navigations (`visitAdminPage` or `page.goto`) and element selections (`page.waitForSelector` or `page.$`) to achieve the same thing.

I think we should instead re-implement it using the approach you do here in gutenberg. However, that would also mean that we'll be losing the coverage of that in our tests. The solution is to add back those here in Core instead, but we don't need to make it a utility, just a simple test.

Basically, swap the code in gutenberg with Core's 😅 . Does it make sense to you? I'll c.c. @tellthemachines for better explanations or other feedbacks 😆 .
